ur_bringup/launch/ur10_bringup.launch

```diff
@@ -11,15 +10,19 @@
   <!-- robot_ip: IP-address of the robot's socket-messaging server -->
   <arg name="robot_ip" doc="IP of the controller" />
   <arg name="reverse_port" default="50001" doc="Port on which controller listens for position values passed to servoj thread from the driver" />
-  <arg name="limited" default="false" doc="If true, limits joint range [-PI, PI] on all joints." />
   <arg name="min_payload"  default="0.0" doc="lower payload bound for the robot" />
   <arg name="max_payload"  default="20.0" doc="upper payload bound for the robot" />
 
+  <!-- robot model parameters files-->
+  <arg name="joints_limits_params" default="$(find ur_description)/config/ur10/joints_limits_parameters.yaml"/>
+  <arg name="physical_params" default="$(find ur_description)/config/ur10/physical_parameters.yaml"/>
+
```

Comment:
Any particular reason not to expose the `visual_params` argument to the `ur..._upload.launch` files?

Reply:
not really (or at least I don't remember why I did that) ... I will change that